OpenVPNHelper/OpenVPNHelper.m

```diff
@@ -84,65 +84,116 @@ - (BOOL)verify:(NSString *)identifier atURL:(NSURL *)fileURL {
     return YES;
 }
 
-- (void)startOpenVPNAtURL:(NSURL *_Nonnull)launchURL withConfig:(NSURL *_Nonnull)config upScript:(NSURL *_Nullable)upScript downScript:(NSURL *_Nullable)downScript leasewatchPlist:(NSURL *_Nullable)leasewatchPlist leasewatchScript:(NSURL *_Nullable)leasewatchScript scriptOptions:(NSArray <NSString *>*_Nullable)scriptOptions reply:(void(^_Nonnull)(BOOL))reply {
+- (void)startOpenVPNAtURL:(NSURL *_Nonnull)launchURL withConfig:(NSURL *_Nonnull)config upScript:(NSURL *_Nullable)upScript downScript:(NSURL *_Nullable)downScript scriptOptions:(NSArray <NSString *>*_Nullable)scriptOptions reply:(void(^_Nonnull)(NSArray*))reply  {
+    
+    
+    
+    
+    NSMutableArray *status = [[NSMutableArray alloc]init];
+    
+    // stores value if the call is successful
+    [status insertObject:[NSNumber numberWithBool: false] atIndex:0];
+    
+    //Store value of any comments
+    [status insertObject:@"Secured Config File" atIndex:1];
+    
+    //Store value of problem type
+    [status insertObject:@"clean" atIndex:2];
```

Comment:
Create a class to return the data. Use enums for states.

Reply:
Hi Johan, in your code, how can I pass data to handler?  because we need to show the list of malicious commands in the alert box?

Reply:
@johankool , have you got the time to look into this?

Reply:
By returning a custom class as I suggested instead of the NSMutableArray.